charts/drax/templates/secret-loki-tenant-override-config.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: reverse-proxy-auth-config
```

Use the usual mechanisms of helm here to guarantee unique names etc with multiple deployments in mind etc.

```suggestion
  name: {{ include accelleran.common.name (dict "top" .) }}-loki-reverse-proxy-auth-config
```

---

This will of course not work with the loki chart as no `tpl` call is present when adding the secret to `extraVolumes`... As the reverse proxy solution that was attempted here isn't maintained anymore, I guess it doesn't matter anymore...